### ISSUE TITLE
Tests

### ISSUE THREAD

**Issue by tashrifbillah** [AMP-SCZ/dpanonymize#5]
Kevin, I don't think you have uploaded the branch you are working on:
```bash
(dpanonymize) [tb571@rc-predict-lochness tests]$ tree
.
├── dpanonymize
│   ├── test_dpanonymize.py
│   └── test_redcap.py
├── dpanonymize_test.py
└── scripts
    └── test_dpanon.py

(dpanonymize1) [tb571@rc-predict-lochness tests]$ pytest -v dpanonymize_test.py
=================================== test session starts ====================================
platform linux -- Python 3.8.10, pytest-6.2.4, py-1.10.0, pluggy-0.13.1 -- miniconda3/envs/dpanonymize/bin/python
cachedir: .pytest_cache
rootdir: dpanonymize
collected 2 items

dpanonymize_test.py::test_test_phoenix_structure_short PASSED                        [ 50%]
dpanonymize_test.py::test_test_phoenix_structure_BIDS PASSED                         [100%]

==================================== 2 passed in 0.60s =====================================
```

**Comment by tashrifbillah** [comment]
> bash dpanonymize_test.sh

This instruction is invalid for the current master.

**Comment by kcho** [comment]
Please run

```
cd dpanonymize/tests
bash dpanonymize_test.sh
```

**Comment by tashrifbillah** [comment]
Okay let me try again--can you point out the `dpanonymize_test.sh` script in the master branch?

**Comment by kcho** [comment]
My bad, just added.

**Comment by tashrifbillah** [comment]
Now I see what you were doing. There are still issues, I shall fix them for us.

**Comment by tashrifbillah** [comment]
Is [this](https://github.com/AMP-SCZ/dpanonymize/blob/fix-tests/setup.py#L11) supposed to install `pytest` when tests are run?

**Comment by tashrifbillah** [comment]
TODO: test a few commands from https://github.com/AMP-SCZ/dpanonymize#how-to-run

**Comment by kcho** [comment]
> Is [this](https://github.com/AMP-SCZ/dpanonymize/blob/fix-tests/setup.py#L11) supposed to install `pytest` when tests are run?

no
https://stackoverflow.com/questions/15422527/best-practices-how-do-you-list-required-dependencies-in-your-setup-py

**Comment by tashrifbillah** [comment]
This is interesting. So before running tests, one should:

> python setup.py test

?

**Comment by kcho** [comment]
I have not built the test to work from the `setup.py` for `dpanonymize`. We should use 

```
cd dpanonymize/tests
bash dpanonymize_test.sh
```

**Comment by tashrifbillah** [comment]
Sorry, I have not been clear--since `pytest` package is missing, your `we should use` won't run the tests.

**Comment by kcho** [comment]
> `pip install pytest` to be able to run the tests.
sounds good
